src/Domain/Creators/TimeDependence/CubicScale.hpp

```diff
@@ -137,24 +123,32 @@ class CubicScale final : public TimeDependence<MeshDim> {
       -> std::vector<std::unique_ptr<domain::CoordinateMapBase<
           Frame::Grid, Frame::Inertial, MeshDim>>> override;
 
-  auto functions_of_time() const -> std::unordered_map<
-      std::string,
-      std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>> override;
+  auto functions_of_time(const std::unordered_map<std::string, double>&
+                             initial_expiration_times = {}) const
+      -> std::unordered_map<
+          std::string,
+          std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>> override;
 
   /// Returns the map for each block to be used in a composition of
   /// `TimeDependence`s.
   MapForComposition map_for_composition() const;
 
+  static std::string name() { return pretty_type::short_name<CubicScale>(); }
```

Comment:
What needs this?

Reply:
ping on this (I think it can be deleted)

Reply:
This is used by `TimeDependenceCompositionTag` in `Domain/Creators/TimeDependence/Composition.hpp` to get the name of the time dependence used in a composition for options. Hence why I only have a `name()` member for `UniformTranslation`, `UniformRotationAboutZAxis`, and `CubicScale` because those are the only three time dependences used in compositions at the moment.

Reply:
I would suggest to replace the code that calls this function with `Options::name<Type>()`, which will use the name of the class unless a `name()` function exists. Then you can just implement a `name()` function for classes where you want to override the default name (if any).

Reply:
I was unaware of this feature, that would be perfect here because currently only `UniformTranslation` needs a custom name. The others can just be the name of the class.